src/main/java/org/terasology/dynamicCities/settlements/SettlementEntityManager.java

```diff
@@ -648,5 +650,8 @@ private boolean checkIfTerrainIsBuildable(Rect2i area) {
         return true;
     }
 
+    public void setCityCyclesBeforeGrowth(int cycles){
```

Comment:
Please add the implication of the _cycles_ here as Javadoc such that users of DynamicCities know what this is about.